Adamant/Models/CoreData/MessageTransaction+CoreDataProperties.swift

```diff
@@ -39,4 +40,7 @@ extension MessageTransaction {
             reactionsData = data
         }
     }
+    func addToRichMessageTransactions(_ transaction: RichMessageTransaction) {
+        self.mutableSetValue(forKey: "richMessageTransactions").add(transaction)
```

Comment:
Oh, I see why you had to do like this. If you mutate richMessageTransactions, nothing will be saved to database.

The same thing would happen if you've used raw NSMutableSet